public/wp-content/plugins/ccs-custom/library/custom-post-types.php

```diff
@@ -249,4 +249,47 @@ function ccs_register_my_cpts() {
     );
 
     register_post_type( "event", $args) ;
+
+
+
+
+    // Digital Brochure(s) content type
```

Comment:
Also, can you move this section before the "event" so the ordering in CMS makes more sense

Reply:
Sure, no problem.